python/lsst/ctrl/bps/wms/panda/idds_tasks.py

```diff
@@ -225,3 +234,12 @@ def get_input_file(self, job_name):
         quantum graph file name
         """
         return next(iter(self.bps_workflow.nodes.get(job_name).get("inputs")))
+
+    def get_pseudo_input_file_name(self, jobname):

```

Comment:
"get" sounds like a lookup.  Perhaps change to something like "create"

Reply:
Also, method needs docstring.

Reply:
Docstring added, function name updated.